Makefile

```diff
@@ -15,8 +15,8 @@ audit:	## run nancy auditor
 
 .PHONY: lint
 lint:
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.2
-	golangci-lint run ./...
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.62.1
```

Comment:
I think the general convention is to install this in the lint.sh script now to prevent unexpected installs.

Reply:
This has now been moved.